# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x86/lilo/activate.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

 Please don't re-enable pseudo.b, it's for testing purposes only and
 requires a full dev86 package (rock includes the bin86 part only).

--- ./Makefile.orig	2003-12-11 16:48:40.000000000 -0600
+++ ./Makefile	2003-12-11 16:49:52.000000000 -0600
@@ -107,7 +107,7 @@
 
 .SUFFIXES:	.img .b .com .S .s
 
-all: lilo bootsect.b disk.com pseudo.b  # $(BOOTS) edit dparam.com activate
+all: lilo bootsect.b disk.com $(BOOTS) edit dparam.com activate  # pseudo.b
 	if [ -x /usr/bin/bcc -o -x /usr/local/bin/bcc ]; then make diagnostic; fi
 
 #
